reference_files/traefik2-k3s-rancher/config/traefik-chart-values.yml

```diff
@@ -50,14 +49,20 @@ env:
       secretKeyRef:
         key: apiKey
         name: cloudflare-apikey-secret
+  - name: CF_API_EMAIL
```

Comment:
interesting, I didn't need this but thank you!

Reply:
I don't know if Traefik & Let's Encrypt need this value for authentication or just the`apiKey`. But since we've already put the email in Kubernetes secret, I just put it there for reference.